### PR TITLE
Fix style crash caused by manually reloading the style

### PR DESCRIPTION
**Pull request by Michael-AirMap**

### CODE REVIEW

File: Source/UI/Views/AirMapMapView.swift [hunk 1]
```diff
@@ -248,19 +265,24 @@ extension AirMapMapView {
...
 			.disposed(by: disposeBag)
 
-		// Reload base style when updating the temporal range or toggling inactive airspace filter
-		Observable.combineLatest(range, showInactiveAirspaceSubject.distinctUntilChanged())
-			.debounce(.milliseconds(100), scheduler: MainScheduler.instance)
-			.subscribe(onNext: { [weak self] (_) in
-				self?.reloadStyle(nil)
+		range
+			.withLatestFrom(mapConfigurables) { ($1.0, $1.1, $1.2, $0) }
+			.observeOn(MainScheduler.asyncInstance)
```
Comment (adolfo) [inline Source/UI/Views/AirMapMapView.swift:270]: How does the async help here?

File: Source/UI/Extensions/MGLMapView+AirMap.swift [hunk 2]
```diff
@@ -165,6 +165,26 @@ extension MGLStyle {
 		return airMapBaseLayers
 	}
 
+	/// Revert layer predicates to their default values to show inative airspace layers
+	func showInactiveAirspace(mapView: AirMapMapView?) {
+		layers
```
Comment (adolfo) [inline Source/UI/Extensions/MGLMapView+AirMap.swift:170]: Feels like the style shouldn't be passed it's owning entity, the map view in this case. Looks like the only reason you're passing it is for the defaultPredicates. Either pass those or move this extension to the map view.

File: Source/UI/Views/AirMapMapView.swift (hunk 1 above)
Comment (Michael-AirMap) [reply Source/UI/Views/AirMapMapView.swift]: That was copied over from the original subscription that has a reentrancy warning. However this subscription does not have that. Removed

File: Source/UI/Extensions/MGLMapView+AirMap.swift (hunk 2 above)
Comment (Michael-AirMap) [reply Source/UI/Extensions/MGLMapView+AirMap.swift]: Good call. Updated

**Pull request merged**